### ISSUE TITLE
Dist folder + packaged CSS and jS

### ISSUE THREAD

**Issue by alvarotrigo** [froala/wysiwyg-editor#1136]
If I want to make use of all plugins at the moment I have to include 17 CSS files and 30 JS ones making a total of 47 HTTP requests and having to write a bunch of code every time I want to use them in other website.

Following the common standard nowadays I suggest making use of a `dist` folder containing all the distributable files and and adding to the current ones the `full` versions of them (packaged CSS and JS files containing all plugins).

You can see this behaviour in popular and complex libraries such as:
- [handsontable.js](https://github.com/handsontable/handsontable/tree/master/dist)
- [select2] (https://github.com/select2/select2)
- [Bootstrap](https://github.com/twbs/bootstrap) (only provides  packaged files, custom ones throw [the website](http://getbootstrap.com/customize/))
- [Mansonry](https://github.com/desandro/masonry) (packaged in the dist folder)
- [Moment.js](https://github.com/moment/moment/tree/develop/min) Providing package with locales.
- [Typeahead](https://github.com/twitter/typeahead.js) (typeahead.bundle.js (bloodhound.js + typeahead.jquery.js)

Tools like [Gulp](http://gulpjs.com/) make it easier nowadays.

##### Expected behavior.
```html
 <link rel="stylesheet" href="dist/froala-editor.full.min.css">
 <script type="text/javascript" src="dist/froala-editor.full.min.js"></script>
````
##### Actual behavior.
```html
  <link rel="stylesheet" href="../css/froala_editor.min.css"/>
  <link rel="stylesheet" href="../css/froala_style.min.css" />
  <link rel="stylesheet" href="../css/plugins/char_counter.css">
  <link rel="stylesheet" href="../css/plugins/code_view.css">
  <link rel="stylesheet" href="../css/plugins/colors.css">
  <link rel="stylesheet" href="../css/plugins/emoticons.css">
  <link rel="stylesheet" href="../css/plugins/file.css">
  <link rel="stylesheet" href="../css/plugins/fullscreen.css">
  <link rel="stylesheet" href="../css/plugins/image.css">
  <link rel="stylesheet" href="../css/plugins/image_manager.css">
  <link rel="stylesheet" href="../css/plugins/line_breaker.css">
  <link rel="stylesheet" href="../css/plugins/quick_insert.css">
  <link rel="stylesheet" href="../css/plugins/table.css">
  <link rel="stylesheet" href="../css/plugins/video.css">

  <script type="text/javascript" src="../js/froala_editor.min.js"></script>
  <script type="text/javascript" src="../js/plugins/align.min.js"></script>
  <script type="text/javascript" src="../js/plugins/char_counter.min.js"></script>
  <script type="text/javascript" src="../js/plugins/code_beautifier.min.js"></script>
  <script type="text/javascript" src="../js/plugins/code_view.min.js"></script>
  <script type="text/javascript" src="../js/plugins/colors.min.js"></script>
  <script type="text/javascript" src="../js/plugins/emoticons.min.js"></script>
  <script type="text/javascript" src="../js/plugins/entities.min.js"></script>
  <script type="text/javascript" src="../js/plugins/file.min.js"></script>
  <script type="text/javascript" src="../js/plugins/font_family.min.js"></script>
  <script type="text/javascript" src="../js/plugins/font_size.min.js"></script>
  <script type="text/javascript" src="../js/plugins/fullscreen.min.js"></script>
  <script type="text/javascript" src="../js/plugins/image.min.js"></script>
  <script type="text/javascript" src="../js/plugins/image_manager.min.js"></script>
  <script type="text/javascript" src="../js/plugins/inline_style.min.js"></script>
  <script type="text/javascript" src="../js/plugins/line_breaker.min.js"></script>
  <script type="text/javascript" src="../js/plugins/link.min.js"></script>
  <script type="text/javascript" src="../js/plugins/lists.min.js"></script>
  <script type="text/javascript" src="../js/plugins/paragraph_format.min.js"></script>
  <script type="text/javascript" src="../js/plugins/paragraph_style.min.js"></script>
  <script type="text/javascript" src="../js/plugins/quick_insert.min.js"></script>
  <script type="text/javascript" src="../js/plugins/quote.min.js"></script>
  <script type="text/javascript" src="../js/plugins/table.min.js"></script>
  <script type="text/javascript" src="../js/plugins/save.min.js"></script>
  <script type="text/javascript" src="../js/plugins/url.min.js"></script>
  <script type="text/javascript" src="../js/plugins/video.min.js"></script>
  <script type="text/javascript" src="../js/languages/ro.js"></script>
```

**Comment by alvarotrigo** [comment]
Also, it makes it difficult for people to reproduce bugs or test the plugin in jsfiddle or codepen. 
Two files makes thing much easier.

**Comment by stefanneculai** [comment]
We created separate plugins so that everyone can include only what he needs. It doesn't make sense to include the video plugin if you never use that feature. We won't change that, because that's the whole idea of having plugins.

**Comment by alvarotrigo** [comment]
I understand your point, having the option to cherry pick what you want is great! Don't get me wrong. And that's why I suggested you to **keep all the plugins separately** and only **additionally** add the packaged files for people who don't want to get bothered with having to remember the plugins name, having to type them all or having to create a GULP task to deal with the concatenation. 

That will only make things easier while keeping the cherry pick functionality. 

I do believe that for testing purposes developers get tired of having to import 47 files or 30 or 20 or even 10! to just quickly test something in jsfiddle, codepen or in their own projects or even to just quickly start an app if you are not bothered for the time-load.

Probably that's the reason why any popular library out there (as big as it might be...) provides packaged files. I got quite surprised to see 47 imports [to "get started" with the library](https://www.froala.com/wysiwyg-editor/docs). (in fact, at first I though it was a joke....)

If Bootstrap had only provided 50 separated files I really doubt it would have become that popular. But instead, providing a [customize feature](http://getbootstrap.com/customize/) that allows developers to cherry pick makes it ideal. Similar to your [customize option](https://www.froala.com/wysiwyg-editor/customize), but allowing you to select the files. 

Personally this puts me off from purchasing the library. I expect a non free product to make things easier for me in all aspects, and this is not one of them... 

I hope you guys can at least consider it a bit more.
